JS.md

```diff
@@ -559,8 +568,10 @@ Any attempts to [`detach`](http://tc39.github.io/ecma262/#sec-detacharraybuffer)
 the detachment performed by [`m.grow`](#webassemblymemoryprototypegrow) shall throw a 
 [`TypeError`](https://tc39.github.io/ecma262/#sec-native-error-types-used-in-this-standard-typeerror)
 
-Return a new `WebAssembly.Memory` instance with [[Memory]] set to `m` and
-[[BufferObject]] set to `buffer`.
+Return a new `WebAssembly.Memory` instance with
+[[Memory]] set to `m`,
+[[BufferObject]] set to `buffer`, and
+[[Maximum]] set to `undefined` if `maximum` is `None` or `maximum` otherwise.
```

Comment:
What's the motivation for storing the maximum here, rather than looking it up in the store?

Reply:
What do you mean "in the store"?

Reply:
He means that the memory instance `m` (living in the Wasm abstract store) already has this information, so the internal slot is redundant.